Libraries/LibLine/Editor.cpp

```diff
@@ -616,7 +616,6 @@ void Editor::handle_read_event()
     }
 
     auto reverse_tab = false;
-    auto ctrl_held = false;
```

Comment:
I can immediately see that I like this change!

Reply:
Then I need to explicitly cast it to int.

Reply:
wrong comment to reply, but fair enough.